File: app/src/test/java/com/booboot/vndbandroid/TestSchedulers.java
package com.booboot.vndbandroid;

import com.booboot.vndbandroid.di.Schedulers;

import org.jetbrains.annotations.NotNull;

import io.reactivex.Scheduler;

public class TestSchedulers implements Schedulers {
    @Override
    public Scheduler io() {
        return io.reactivex.schedulers.Schedulers.trampoline();
    }

    @Override
    public Scheduler ui() {
        return io.reactivex.schedulers.Schedulers.trampoline();
    }

    @NotNull
    @Override
    public Scheduler newThread() {
        return io.reactivex.schedulers.Schedulers.trampoline();
    }
}

File: app/src/test/java/com/booboot/vndbandroid/TestSchedulers.kt
package com.booboot.vndbandroid

import com.booboot.vndbandroid.di.Schedulers

import io.reactivex.Scheduler

class TestSchedulers : Schedulers {
    override fun io(): Scheduler {
        return io.reactivex.schedulers.Schedulers.trampoline()
    }

    override fun ui(): Scheduler {
        return io.reactivex.schedulers.Schedulers.trampoline()
    }

    override fun newThread(): Scheduler {
        return io.reactivex.schedulers.Schedulers.trampoline()
    }
}